src/test/java/org/ohdsi/authenticator/service/RestAuthenticationTest.java

```diff
@@ -29,30 +28,31 @@
     @Value("${credentials.rest-atlas.password}")
     private String atlasPassword;
 
+    @Autowired
+    protected JwtTokenProvider jwtTokenProvider;
+
     @Test
     public void testRestArachneAuthSuccess() {
 
         final var method = "rest-arachne";
         var authRequest = new UsernamePasswordCredentials(arachneUsername, arachnePassword);
         UserInfo userInfo = authenticator.authenticate(method, authRequest);
-        Assert.isTrue(
-                Objects.equals(userInfo.getUsername(), authRequest.getUsername())
-                && getExpirationInSecs(userInfo.getToken()) >= jwtTokenProvider.getDefaultValidityInSeconds(),
-                "Failed to authenticate user with proper credentials"
-        );
+
+        String accessToken = userInfo.getToken();
+        Assert.assertEquals("Failed to authenticate user with proper credentials", authRequest.getUsername(), userInfo.getUsername());
+        Assert.assertTrue("Failed to authenticate user with proper credentials", getExpirationInSecs(accessToken) >= jwtTokenProvider.getValidityInSeconds());
+
     }
 
     @Test
     public void testRestAtlasAuthSuccess() {
 
-        final var method = "rest-atlas";
-        var authRequest = new UsernamePasswordCredentials(atlasUsername, atlasPassword);
+        final String method = "rest-atlas";
+        UsernamePasswordCredentials authRequest = new UsernamePasswordCredentials(atlasUsername, atlasPassword);
```

Comment:
Please, let's not change the code back and forward. Could you either provide arguments against var and, if agreed, remove ALL var-s, or follow existed coding practice

Reply:
done